<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OnlineForm</class>
 <widget class="QWidget" name="OnlineForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>684</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="checkUserForNextStep">
       <property name="text">
        <string>Ожидать команды пользователя на продолжение.</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="openScenarioButton">
       <property name="text">
        <string>открыть файл сценария</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>старт</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="iterationsLabel">
         <property name="text">
          <string>Повторений*</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="iterationsBox">
         <property name="minimum">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="reversAcquisitionBox">
         <property name="text">
          <string>менять направление сценария</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QProgressBar" name="stepProgressBar">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="finishOnThisIterationBox">
       <property name="text">
        <string>сделать эту итерацию последней</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pauseButton">
         <property name="text">
          <string>пауза</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resumeButton">
         <property name="text">
          <string>продолжить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="text">
          <string>стоп</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QTextBrowser" name="infoBrowser"/>
   </item>
   <item row="3" column="0">
    <widget class="QListWidget" name="scenarioView">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="showDropIndicator" stdset="0">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="commentEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="sendComment">
       <property name="text">
        <string>отправить комментарий</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Информация:</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="3" column="0">
        <widget class="QLabel" name="operatorSurnameLabel">
         <property name="text">
          <string>Фамилия оператора*</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="operatorSurnameEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Комментарии по набору</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QPlainTextEdit" name="acquisitionCommentsBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="sessionLabel">
         <property name="text">
          <string>Сеанс*</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="sessionEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="groupLabel">
         <property name="text">
          <string>Название групы*</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="groupEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0">
    <widget class="QTextBrowser" name="paramsBrowser"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="metadataLabel">
     <property name="text">
      <string>Информация о наборе</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
